docs/v1/P4Runtime-Spec.mdk

```diff
@@ -1257,7 +1257,10 @@ The `ActionProfile` message includes the following fields:
   groups for an Action Selector.
 
 * `max_group_size`, an `int32` representing the maximum number of *weighted*
-  member entries in any given Selector group, or 0 for an Action Profile.
+  member entries in any given Selector group, or 0 for an Action Profile. PSA
```

Comment:
I may be confused here, but it seems that here it is saying that max_group_size is the maximum _number_ of member entries, whereas later it is saying that max_group_size is the maximum total weight of all member entries.

Is it sometimes one, sometimes another?  In which cases?

Reply:
It is always meant to be the sum of member weights, but I guess "weighted member entries" doesn't make this clear, so I made a bunch of edits to be more consistent.

Reply:
PTAL.

Reply:
This might be a stupid question, but why is it the sum of weights rather than the count?  I assumed the hardware stores each member + weight, so the limit would be regardless of the value of the weight.

Reply:
Not a stupid question at all.  I do not claim knowledge of all hardware implementations of weighted cost multipath load balancing, but at least one common way is to implement equal cost multipath load balancing in hardware, with one hardware entry for each equal cost member, and then use the weights to determine how many times to duplicate each weighted entry in the hardware table, so that you achieve the relative frequencies specified by the weights.  Thus the hardware table size can end up containing "sum of weights" number of entries rather than "sum of number of entries".

Reply:
Got it, thanks for the explanation.